qualification/system_module_config.json

```diff
@@ -1,29 +1,92 @@
 {
   "modules": {
-    "pass": {
+    "bacext": {
       "enabled": true
     },
-    "nmap": {
+    "bacnet": {
+      "enabled": true
+    },
+    "brute": {
       "enabled": true
     },
     "macoui": {
-     "enabled": true
+      "enabled": true
     },
-    "switch": {
+    "mudgee": {
       "enabled": true
     },
-    "brute": {
+    "nmap": {
       "enabled": true
     },
-    "bacext": {
+    "switch": {
```

Comment:
@InBrewJ Do we need to add the PoE stuff in here too?

Reply:
@pisuke Done. Do we need to add something about poe tests into the required Pass/fail section too?

Reply:
Yup, good catch, please proceed adding this too, thanks!

Reply:
OK, I've added all the configuration bits for the existing tests.